web/app/controllers/resources/RdfConverter.java

```diff
@@ -47,9 +45,11 @@ public String getName() {
 	 */
 	public static String toRdf(final String jsonLd, final RdfFormat format) {
 		try {
-			final Object jsonObject = JsonUtils.fromString(jsonLd);
-			final JenaTripleCallback callback = new JenaTripleCallback();
-			final Model model = (Model) JsonLdProcessor.toRDF(jsonObject, callback);
+			//convert json-ld string into InputStream as is required by the read() function.
+			InputStream targetStream = new ByteArrayInputStream(jsonLd.getBytes());
+			Model model = ModelFactory.createDefaultModel() ;
```

Comment:
Other variables here are declared `final`, not sure if it's an intended change.

Reply:
Not intended. IDE didn't take care and I forgot about it. Fixed. Thx.